extensions/cornerstone/src/services/SegmentationService/SegmentationService.ts

```diff
@@ -1241,37 +1241,39 @@ class SegmentationService extends PubSubService {
 
     const { fillAlpha } = this.getConfiguration(toolGroupId);
 
-    let count = 0;
-    const intervalTime = 16;
-    const numberOfFrames = Math.ceil(animationLength / intervalTime);
+    let startTime: number = null;
+    const animation = (timestamp: number) => {
+      if (startTime === null) {
+        startTime = timestamp;
+      }
+
+      const elapsed = timestamp - startTime;
```

Comment:
What do the changes here have to do with this change?

Reply:
Nothing I saw some performance degradation on highlight SEG and I fixed it